### PR TITLE
Release PR v2.2.9

### PR DESCRIPTION
**Pull request by sapthasurendran**
Signed-off-by: sapthasurendran <saptha.surendran@ibm.com>

### CODE REVIEW

File: release_notes/v2.2.9.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a maintenance release containing bug fixes.
```
Comment (bestbeforetoday) [inline release_notes/v2.2.9.txt:1]: Right now I'm generally putting text in here that matches what will go in the release description when it is created in GitHub. Look here for examples:
- https://github.com/hyperledger/fabric-sdk-node/blob/release-2.2/release_notes/v2.2.8.txt
- https://github.com/hyperledger/fabric-sdk-node/releases

I am mentioning significant changes or fixes that will be of interest to users, which for this release should probably be the fix to reconnect of event listening:
- https://github.com/hyperledger/fabric-sdk-node/pull/481

**Review by bestbeforetoday — CHANGES_REQUESTED** [review]
The release PR needs to be raised against the release branch, which in this case is _release-2.2_, not _main_.

**Comment by bestbeforetoday** [comment]
I think you'll need to rebase just your changes on _release-2.2_. There should just be a single commit rather than the 30 commits you currently have!

**Comment by sapthasurendran** [comment]
PR raised from wrong branch .
New PR raised #486

**Pull request closed without merging**